WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -91,6 +91,7 @@ public async Task<bool> StartRoundAsync(RoundState roundState, CancellationToken
 
 			// Register coins.
 			await RegisterCoinsAsync(aliceClients, cancellationToken).ConfigureAwait(false);
+			roundState = await RoundStatusUpdater.CreateRoundAwaiter(roundState => roundState.Phase == Phase.ConnectionConfirmation, cancellationToken).ConfigureAwait(false);
```

Comment:
should not wait as long as `cancellationToken`, but with a linked token that has some timeout